hdk-rust/wasm-test/src/lib.rs

```diff
@@ -167,6 +168,47 @@ zome_functions! {
     }
 }
 
+zome_functions! {
+    check_query: | | {
+
+        // Query DNA entry
+        let result = hdk::query(&EntryType::Dna.to_string(), 0);
+        assert!(result.is_ok());
+        assert!(result.unwrap().len() == 1);
```

Comment:
This way, we don't know why the test failed. Better do something like:

```rust
let query_result = hdk::query(&EntryType::Dna.to_string(), 0).expect("blah blah blah");
assert!(query_result.len() == 1);
```

Does this even work with wasm?

Reply:
Not really. You get a "Trap: Unreachable" in any case.
I made a zome_assert! macro but it doesn't work well inside the zome_functions! macro for now.